src/logic/safe/transactions/gas.ts

```diff
@@ -120,18 +141,16 @@ const getGasEstimationTxResponse = async (txConfig: {
   try {
     const result = await web3.eth.call(txConfig)
 
-    // GETH Nodes
+    // GETH Nodes (old version)
     // In case that the gas is not enough we will receive an EMPTY data
-    // Otherwise we will receive the gas amount as hash data
+    // Otherwise we will receive the gas amount as hash data -> this is valid for old versions of GETH nodes
```

Comment:
I think this will become legacy code with time, but instead of calling it "old versions" or "new GETH nodes" can we be more specific about the GETH version?

beautifully commented code, BTW

Reply:
Good idea, I updated the comment :)